README.md

```diff
@@ -8,10 +8,14 @@ Compatibility
 The following Grove compatible devices are supported in Python:
 
 * Dexter Industries:
-  * Distance Sensor
-  * Inertial Measurement Unit Sensor
-  * Light Color Sensor
-  * Temperature Humidity Pressure Sensor
+  * **[Distance Sensor](https://www.dexterindustries.com/shop/distance-sensor/)** - Distance Sensor for the GoPiGo.  The distance sensor can be mounted to the GoPiGo Raspberry Pi robot using the servo package to enable rotation.  It detects distances from obstacles and objects, giving your robot the ability to navigate.
```

Comment:
It's not limited to the GoPiGo. It can also be used by BrickPi and GrovePi .

Reply:
*can be mounted to the GoPiGo Raspberry Pi robot using the servo package to enable rotation* 
-> can be mounted to the GoPiGo Rasbperry Pi robot with or without the servo package to enable rotation

I'm not sure how to end that sentence, to be honest, but I wouldn't want to give the impression the servo is required to mount the distance sensor